helpers/helper-translation-discussion.php

```diff
@@ -253,9 +253,11 @@ public function rewrite_original_post_type_permalink( string $post_link, WP_Post
 		if ( ! $project ) {
 			return $cache[ $post->ID ];
 		}
+		$locale_slug = isset( $_POST['meta']['locale'] ) ? $_POST['meta']['locale'] : null;
```

Comment:
This indeed needs at least a `sanitize_form()` or better a check against out list of locales.